.github/workflows/tests.yaml

```diff
@@ -14,98 +14,74 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Add local bin to PATH
-        run: |
-          echo "${HOME}/.local/bin" >> $GITHUB_PATH
       - name: Install python packages and dependencies
         run: |
           sudo apt update
-          sudo apt install -y python3-pip python3-venv libapt-pkg-dev
-          python3 -m venv ${HOME}/.venv
-          source ${HOME}/.venv/bin/activate
-          pip install -U pip wheel setuptools
-          pip install -U -r requirements.txt -r requirements-dev.txt -r requirements-focal.txt
-          pip install -e .
+          sudo apt install -y libapt-pkg-dev
+          pip install -U \
+                         https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-distutils-extra/2.43/python-distutils-extra_2.43.tar.xz \
```

Comment:
distutils-extra wasn't in `requirements-focal.txt` but I assume it's required since you added it here.

Reply:
it is a dependency of python apt unfortunately and we want the right one